### ISSUE TITLE
MMD file support, similar or better than the one in threejs

### ISSUE THREAD

**Issue by blurymind** [BabylonJS/Babylon.js#6417]
# Feature request
Threejs has this gem:
https://threejs.org/docs/#examples/loaders/MMDLoader
https://threejs.org/docs/#examples/animations/MMDPhysics
https://threejs.org/docs/#examples/animations/MMDAnimationHelper

demo
https://threejs.org/examples/webgl_loader_mmd.html

Was wondering if we can get it in babylonjs. With the huge amounts of community made content, it could make for some interesting web apps. The MMD content tends to be shared as video on the web, having it be shared in a more interactive way would be better.

It would be amazing if we can get runtime access to the skeleton, animation, model parts, materials, etc - be able to change them dynamically 

I believe  it is based on this:
http://takahirox.github.io/mmd-viewer-js/
https://github.com/takahirox/mmd-viewer-js
but I might be wrong

The best thing about this is really how when loaded we get physics, animation and materials working out of the box. They are all set in the file.

Threejs has made this super easy.
So to summarize the selling points:
- big amount of free quality content with community making it actively
- little effort to load an animated model with physics set on it (traditionally a pain in the neck with other formats). Its all in the file
- the content sort of works great for interactive/vr apps - it compliments that technology
- would make for some fun babylonjs demos : ) (even Unity3d got one)

**Comment by deltakosh** [comment]
Hello! we use the forum for preliminary discussion place :)
Can you repost your request there?
forum.babylonjs.com

Thanks a lot!
